src/test/java/de/hhu/propra/link/services/AbbreviationServiceTest.java

```diff
@@ -0,0 +1,25 @@
+package de.hhu.propra.link.services;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class AbbreviationServiceTest {
+
+    AbbreviationService abbreviationService;
+
+    @BeforeEach
+    void setup() {
+        this.abbreviationService = new AbbreviationService();
+    }
+
+    @Test
+    void testAbbreviationLong() {
+        String urlString = "https://www.sub.example.com/path/to/index.html";
+        Optional<String> result = abbreviationService.generateAbbreviation(urlString);
+        assertThat(result).contains("sbxmplpti");
```

Comment:
```suggestion
        assertThat(result).isEqualTo("sbxmplpti");
```

Contains würde auch substrings erlauben. Hier ist das Ergebnis aber ganz klar bekannt

Reply:
This is correct, result is an `Optional<T>` and thus responds to `contains(...)`